xep-0388.xml

```diff
@@ -15,11 +15,31 @@
   <sig>Standards</sig>
   <dependencies>
     <spec>XMPP Core</spec>
+    <spec>rfc5802</spec>
+    <spec>XEP-0440</spec>
   </dependencies>
   <supersedes/>
   <supersededby/>
   <shortname>sasl2</shortname>
   &dcridland;
+  &tmolitor;
+  &mwild;
+  <revision>
+    <version>0.4.0</version>
+    <date>2022-10-06</date>
+    <initials>tm</initials>
+    <remark>
+      <ul>
+        <li>Bump namespace</li>
+        <li>Make SCRAM hashes (and other SASL mechanism families) upgradable</li>
+        <li>Add dependency on &xep0440; and &rfc5802;</li>
+        <li>Update security considerations and business rules</li>
+        <li>Clarify &lt;continue/> and tasks</li>
+        <li>Add expansion point to inline stream resumption and BIND2 (and possibly others)</li>
```

Comment:
To be clear `<inline/>` is merely a place where SASL 2 extensions can announce themselves (instead of being direct stream features).
But I don’t have any strong feelings on whether stream feature or inline stream feature is the cleaner approach. One signals that it depends on SASL2 the other one would allow use to use Bind 2 over Sasl 3.

Reply:
SASL3 could well define an extension point, too. I don't think it's neccessary to remove the `<inline/>` just because of an hypothetical SASL3.

Reply:
I think this is already discussed on-list. I'll mark that as resolved here for now.